website/docs/components/modal/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,352 @@
+
+- To main consistency the only property that should be changed in the actions within the footer is the text property. An exception to this is in the case of a Modal with critical or destruction function, in which the primary button should be changed to **critical**.
+- Action buttons should remain consistent with the variants defined in the number of actions in the footer; **primary**, **secondary**, and **tertiary**, in that order.
+- Action buttons should not use an icon within the context of the Modal. Pairing an icon with text in a button is meant to more clearly outline the button function, in the context of a Modal the function should be apparent through the use of other elements like **title**, **tagline**, and **icon**.
+- Action buttons should use size **medium** to maintain hierarchy and visual weight.
+
+#### One action
+
+<Hds::Modal::Footer>
+  <Hds::ButtonSet>
+    <Hds::Button @text="Confirm" />
+  </Hds::ButtonSet>
+</Hds::Modal::Footer>
+
+
+#### Two actions
+
+<Hds::Modal::Footer>
+  <Hds::ButtonSet>
+    <Hds::Button @text="Confirm" />
+    <Hds::Button @text="Cancel" @color="secondary" />
+  </Hds::ButtonSet>
+</Hds::Modal::Footer>
+
+#### Action best practices
+
+- Most Modals should use two inverse actions; or two actions which perform opposite functions (i.e. “ok/cancel”, “yes/no”).
+- In the case of a single action, the dismiss acts as an escape hatch if the user needs to return to the main page content.
+  - Depending on the information being requested a secondary action that closes or cancels the model may not perform the same function as dismissing the Modal. For example; responding “no” to a “yes/no” question is not the same as dismissing the Modal, the latter of which does not submit a response.
+  - While certain experiences may only call for a single action button, most Modal instances should have a “cancel” button as a more explicit method to close the Modal.
+
+#### Three actions
+
+<Hds::Modal::Footer>
+  <Hds::ButtonSet>
+    <Hds::Button @text="Confirm" />
+    <Hds::Button @text="Cancel" @color="secondary" />
+    <Hds::Button @text="Tertiary" @color="tertiary" @iconPosition="leading" @icon="plus" />
+  </Hds::ButtonSet>
+</Hds::Modal::Footer>
+
+##### Tertiary best practices
+
+- If a tertiary action is necessary it should not be grouped with the button set of primary and secondary actions.
+  - Button grouping should be determined by functionality; since the primary and secondary buttons are performing related (inverse) functions they are grouped. In the case of a tertiary button within a Modal, it’s function is generally unrelated to the primary and secondary actions and therefore shouldn’t be grouped in the same way.
+- Common examples of tertiary actions include:
+  - Linking to external documentation
+  - Linking to another related area or function within the application; best used to help the user better understand or collect the information needed to interact with the Modal.
```

Comment:
```suggestion
    - Linking to external documentation
    - Linking to another related area or function within the application; best used to help the user better understand or collect the information needed to interact with the Modal.
```

I think this will fix the list indentation